web/static/js/components/surveys/SurveyCard.jsx

```diff
@@ -74,54 +75,80 @@ class SurveyCard extends Component<any> {
       {survey.description}
     </div>
 
+    const hideSurveyCard = !inPanelSurveyFolder && survey.isPanelSurvey && !panelSurvey
```

Comment:
This is for not showing surveys in in the survey index? 
I'm a bit confused with this

Reply:
Shouln't the index ignore those surveys?

Reply:
Yeah, probably. I'll try to do it this way! 🙃